profiler/shakesapp/shakesapp/shakesapp.pb.go

```diff
@@ -0,0 +1,210 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Probably need to add a license exception to https://github.com/GoogleCloudPlatform/golang-samples/blob/master/license_test.go#L42.